src/Controller/Speaker/Create.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Controller\Speaker;
+
+use App\Entity\Speaker;
+use App\Form\SpeakerType;
+use App\Repository\SpeakerRepository;
+use App\Service\FileUploader;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\RouterInterface;
+use Twig_Environment;
+
+final class Create
+{
+    private $renderer;
+    private $speakerRepository;
+    private $formFactory;
+    private $router;
+    private $fileUploader;
+
+    public function __construct(
+        Twig_Environment $renderer,
+        SpeakerRepository $speakerRepository,
```

Comment:
Never ever set your implementation of your repository here, this should be an interface as specific as possible. Have a look at https://www.tomasvotruba.cz/blog/2017/10/16/how-to-use-repository-with-doctrine-as-service-in-symfony/ and https://matthiasnoback.nl/2014/05/inject-a-repository-instead-of-an-entity-manager/ (interfaces from the second, repository as a service using composition as in the first).